packages/cli/src/commands/logs/command.ts

```diff
@@ -0,0 +1,66 @@
+import { Command } from '../help';
+import { getPkgName } from '../../util/pkg-name';
+
+export const logsCommand: Command = {
+  name: 'logs',
+  description: 'Print vercel logs for a deployment.',
```

Comment:
I don't think we need to say "vercel" and it's nice to say "specific" as it implies there's an argument required.

```suggestion
  description: 'Display logs for a specific deployment.',
```